pom.xml

```diff
@@ -158,5 +158,9 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Move it to the <!-- Compilation dependencies --> block

Reply:
done